lua/neotest-kotlin/kotest-treesitter-query.lua

```diff
@@ -1,46 +1,39 @@
-local TreesitterQuery = {}
-
--- These are treesitter queries for pulling data of of the AST,
+-- These are treesitter queries for pulling data out of the AST,
 -- More information on how this works over here: https://neovim.io/doc/user/treesitter.html
--- And you can interactively play around with these kotlin queries here: https://fwcd.dev/tree-sitter-kotlin/
-
-TreesitterQuery.value = [[
+-- And you can interactively play around with these kotlin queries here: https://fwcd.github.io/tree-sitter-kotlin/
```

Comment:
Little bit going on here, but the URL that was here before (https://fwcd.dev/tree-sitter-kotlin/) was broken and is now https://fwcd.github.io/tree-sitter-kotlin/